python_modules/dagster/dagster/_core/definitions/asset_graph_differ.py

```diff
@@ -55,7 +55,7 @@ class ValueDiff(Generic[T]):
     new: T
 
 
-@whitelist_for_serdes
+@whitelist_for_serdes(kwargs_fields={"added_keys", "changed_keys", "removed_keys"})
 @record
 class DictDiff(Generic[T]):
```

Comment:
we should probably just bail back to `NamedTuple` until we can fix `@record` w/ `Generic` on py38

Reply:
https://github.com/dagster-io/dagster/pull/24751